src/interfaces/subgraphTypes.ts

```diff
@@ -18,413 +18,236 @@ export enum OrderStatus {
-  totalOI?: string;
-
-  // " Total Open Interest (assets) "
-  totalOIAssets?: string;
-
-  // " Accumulated OI USD at position avgPrice for Longs "
-  accumulatedOILongs?: string;
-
-  // " Accumulated OI USD at position avgPrice for Longs "
-  accumulatedOIShorts?: string;
-
-  // " If closeOnlyMode is true, no new positions can be opened. Only existing position can be closed "
-  closeOnlyMode?: boolean;
-
-  // " Timestamp when the cumulative fees were last updated "
-  feeLastUpdatedTimestamp?: string;
-
-  //  " Price deviation for Longs "
-  priceDeviationLongs?: string;
-
-  // " Price deviation for Shorts "
-  priceDeviationShorts?: string;
-
-  //  " Market utilization for Longs "
-  utilizationLongs?: string;
-
-  // " Market utilization for Shorts "
-  utilizationShorts?: string;
-
-  // " Market skew for Longs "
-  marketSkew?: string;
+  /** Unique identifier for the market */
+  id: string;
+  
+  /** Name of the market */
+  name: string;
 
-  // " Cumulative Base Fees for Longs "
-  baseFeeCumulativeLongs?: string;
+  /** Symbol representing the market */
+  symbol: string;
 
-  //  " Cumulative Base Fees for Shorts "
-  baseFeeCumulativeShorts?: string;
+  /** Total size of the market */
+  size: string;
 
-  // " Cumulative Dynamic Fees for Long "
-  dynamicFeeCumulativeLongs?: string;
+  /** Skew value of the market */
+  skew: string;
 
-  //  " Cumulative Dynamic Fees for Shorts "
-  dynamicFeeCumulativeShorts?: string;
+  /** Current funding rate of the market */
+  currentFundingRate: string;
 
-  // " Deviation Coefficient for liquidity curve "
-  deviationCoeff?: string;
+  /** Current funding velocity of the market */
+  currentFundingVelocity: string;
 
-  // " Deviation constant for liquidity curve "
-  deviationConst?: string;
+  /** Feed ID for price oracle */
+  feedId: string;
 
-  // " Base coefficient for base borrowing fees "
-  baseCoeff?: string;
+  /** Maximum funding velocity allowed for the market */
+  maxFundingVelocity: string;
 
-  // " Base constant for base borrowing fees "
-  baseConst?: string;
+  /** Skew scale of the market */
+  skewScale: string;
 
-  // " Maximum Dynamic borrowing fees "
-  maxDynamicBorrowFee?: string;
+  /** Fee charged for market maker transactions */
+  makerFee: string;
 
-  // " Dynamic coefficient for Dynamic borrowing fees "
-  dynamicCoeff?: string;
+  /** Fee charged for market taker transactions */
+  takerFee: string;
 
-  // " Market transaction creation hash "
-  transactionHash?: string;
+  maxMarketValue:string,
 
-  // " Address of market creator "
-  senderAddress?: string;
+  maxOpenInterest:string,
 
-  // " Pyth Price Id "
-  pyth?: PythData;
+  interestRate:string
 }
 
+
 ////////////////////////////////////////////////////////////////
 //////////////////////    ORDERS    ////////////////////////////
 ////////////////////////////////////////////////////////////////
-
+// settlementReward, referralFees, partnerAddressts
 export interface Order {
```

Comment:
Mark all fields as optional for all responses in subgraph types

Reply:
Done